### PR TITLE
96 add sia clutering to pre processing

### PR DESCRIPTION
**Pull request by Heaghneyn**

### CODE REVIEW

**Comment by mcuadera** [comment]
@Heaghneyn what's the best way to test this out without running the entirety of preprocessing?

**Comment by Heaghneyn** [comment]
that's a good question and why I didn't provide instructions, would like to discuss with the dev team

File: R/utils.R
[the remark this answers is not in the document]
Comment (Heaghneyn) [reply R/utils.R:1]: you're right there is no point in assigning outputs to the run_cluster_dates and we can just feed vax types into a loop to run_cluster_dates. 

During dev call can you talk me through your thoughts on years and params?

**Review by mcuadera — APPROVED** [review]
Recommend rewrite to: 
```
for(i in vax.types) {
    sia.01 |>
      run_cluster_dates(min_obs = 4, type = i)
  }
```
Otherwise, approved of the changes

**Pull request merged**